server/activity_manager.go

```diff
@@ -119,7 +119,7 @@ func (m *ActivityManager) getWebhookContext(ctx context.Context, activity *api.A
 			title = "Issue canceled - " + meta.issue.Name
 		}
 	case api.ActivityIssueCommentCreate:
-		title = "Comment created"
+		title = "Comment created - " + meta.issue.Name
```

Comment:
Use fmt.Sprintf() and below. It's easier to read the string format than string construction.

Reply:
Done.